### PR TITLE
Adding force: and from: template parameters

### PR DESCRIPTION
**Pull request by fyyyyy**
```
---
force: true => will overwrite file without question
from: generator/action/template.name => will load template body from another file in template folder. Useful for sharing templates between generators.
---
```

#226

### CODE REVIEW

**Comment by chiefchief** [comment]
not working, still asking

**Comment by chiefchief** [comment]
not working, still asking

**Comment by fyyyyy** [comment]
hm thats strange, it seems to work for me. maybe write access is missing for the folder?

**Comment by systwo** [comment]
The _from_ feature doesn't seem to process and render the EJS content i.e. `<%= prop %>` in the target _from_ file is output verbatim with the tags rather than the `prop` variable value being output

**Comment by fyyyyy** [comment]
Oh yes, its missing a call to render the `from:` body

renderTemplate(
            body,
            { ...args, attributes: renderedAttrs },
            config,

File: src/ops/add.ts [hunk 1]
```diff
@@ -39,11 +39,17 @@ const add = async (
     }
   }
 
+  if (from) {
+    const from_path = path.join(args.templates, from)
+    const file = fs.readFileSync(from_path).toString()
+    action.body = file
```
Comment (fyyyyy) [inline src/ops/add.ts:45]: missing a call to render body

**Pull request merged**